src/test/java/systemtests/DeleteCommandSystemTest.java

```diff
@@ -30,9 +29,11 @@
     public void delete() {
         /* ----------------- Performing delete operation while an unfiltered list is being shown -------------------- */
 
-        /* Case: delete the first person in the list, command with leading spaces and trailing spaces -> deleted */
+        /* Case: delete the first person in the list with mixed-case,
```

Comment:
Will it be better to create a separate new test case for testing mixed-case command instead of replacing the default test case?

As it stands, there is no test for the constant value ```DeleteCommand.COMMAND_WORD```.

Reply:
Yep! Can do that